build/premake5.lua

```diff
@@ -11,7 +11,11 @@ project "rive"
 
     files {"../src/**.cpp"}
 
-    buildoptions {"-Wall", "-fno-exceptions", "-fno-rtti", "-Werror=format"}
+    buildoptions {"-Wall", "-fno-exceptions", "-fno-rtti"}
+
+    if os.host() ~= "windows" then
+        buildoptions {"-Werror=format"}
```

Comment:
I always wonder if we should just enable this everywhere...probably yes?

Reply:
I used `premake5 vs2019` to generate a Visual Studio solution, then tried to compile.

`-Werror=format` threw an error and I had to remove it in order to compile. Both `-fno-exceptions` and `-fno-rtti` are ignored by visual studio:

```
1>------ Build started: Project: rive, Configuration: release x64 ------
1>cl : command line warning D9002: ignoring unknown option '-fno-exceptions'
1>cl : command line warning D9002: ignoring unknown option '-fno-rtti'
```